### ISSUE TITLE
[Bug] line.select.lineStyle 不起作用

### ISSUE THREAD

**Issue by zmcode** [apache/echarts#19194]
### Version

5.4.3

### Link to Minimal Reproduction

https://echarts.apache.org/examples/zh/editor.html?code=PYBwLglsB2AEC8sDeAoWsAeBBDEDOAXMmurGAJ4gCmRARAMYCGYVA5sAE7m0A0J6AE2aMiAbVoBZGL1i0AKgFcqM2gHUqAlXIAWClQDEOEFQGVmphdFoBdEgF8-6cjnxEkDknipGqhWKNRSWCEwEX8ADgAmAAYeWABOAGZIuPjogEZUxIAWOPTItLzExNjYdOTo60dSCmo6ABsIaGVq9C96qnowN35SRuaTCg6eoKChPG0AeQAzaa9u2Eil1tH6YHrOOg5WACNGAAp0zLKANhO4yOzcsoBKXl6ggHcIATBtImLilaDQRnoIChEaIAOni8W-pAmjAEwEeAGF1ptZNs9vslsdIsk4uE8ncIW1tNDYTM5lQwAANIjLB6Qwkwx4k-YATQ-Xxp6AAtgDvAAZCBchZLFLs2BQ-kAIXqCg4VJiNI8o1gjA4VEYg3Iw2IivQaw2MuRuwOMVKkQxx3Sd3lEO5HPVmsC2t1SNoKIO6QArBdIp7Fj6LbR5b0FaRlaq7TQtatEfqXYbDvFzrBsikkyn_UGVu1OiwBFIBBHaF4fHh7kEqByQIS8K5I0F-lRwyNHdGtnH0onIgB2C5XXEBxV2IP2Wx2IA&_source=echarts-doc-preview

### Steps to Reproduce

1. 选中标记, 折线图已经给选中了, 但是配置当中的, 线的颜色并不会变化, 还有areaStyle的颜色也不生效

### Current Behavior

line.select lineStyle和 areaStyle不起作用

### Expected Behavior

希望选中标记的时候, 能改变选中的折线图颜色和大小, 以及areaStyle的颜色

### Environment

```markdown
- OS:
- Browser:116.0.5845.188
- Framework:
```

**Comment by helgasoft** [comment]
yes, **serious bug** confirmed for **series-line.select** - only _itemStyle_ works, _lineStyle_ and _areaStyle_ do not.
workaround with event [selectchanged](https://echarts.apache.org/en/api.html#events.selectchanged) - [Demo Code](https://echarts.apache.org/examples/en/editor.html?code=DYZwvABA3hDuCWATALgCwFwCYA0EDGA9sAQE7oDkARsAK4Cm5EAvgFACG40-Rp6E5JAOaU2ACgCMATgBsuACw4IC3OICU5VgQAOyeAQB2ESFBYQIADwCC5-CD4mzZ5AE8tdPuTxtkdQaWfk2KaOiN5sfADa5ACyBoH8ACr08eQA6nSIKQmoNCkAYiTwKQDK3iU0-uQAusFMQWbO1rb2dcEgdIV0dhARwWYOjmahyOE9AByYAAy4kgDMipKT4jOzciqYiyqzs9MQ4vOTVfWDEC5uHsDw-gzHgyDOALaURMXwAF7uEJg4fY6X18UXMBPqBbo42CQ6GxAc5gXwOGCzO1gXgfIhYohPuR2p0QIFfmY6A8tKgOM1oAS_lc6DC4VwECgMBAxrhCMQyPwhCIJLIvgB2XCYORrPbqZiU1onCDIuio-yUszwHwPWmfAZSsxs3ic4RicQAVkFmENXxNanICvFUsljlYZhqTAA3CwQFdIPoaMBgM6Hs4AMKkkjIAB0BlE2LoKOQeFJ-kEGXioi0qiMAD4KYqAGYQUSu_QAQjAHq9KZgZgA9OXIe1kL9tLoDMGcfAuhE81VgxCoarIBxnYN63p9E2Oi2QG2rh3_jSgXRIKBfnn3Z7vbUIJH2hnHHmjBAtCOoxkIocR7iAJL6THmfuOQeN5ut9vB6eq3fqkIcVAAeUzmZrfG-H4pS1DkBF1CRlj2aReSFEVzUROAkDQPhtgQ7Q2DwJVnD4SZg0kSQEJAUlEAIWA_R4UCuTEb5IMweZcBZUV8SlIi2BI2Afz_OhkAADQAoCTlY9jOJrABNFDUMpB4lQ6AAZeBpOQfiBLuYjSIAIVoDlvkmX4nTrHQh1PMcJ30Dsu2hWc3xAi54EEVBkGoZJ9LMO0IF9AMIRDGsv0MsM730VRnSYIKgA)

**Comment by zmcode** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
